app/src/Components/DisplayCards/DisplayCards.tsx

```diff
@@ -15,23 +14,40 @@ function DisplayCards ({ search, sortBy, order }: Props) {
   const [pageNumber, setPageNumber] = useState(0);
   const [cards, setCards] = useState<PokemonCard[]>([]);
   const [totalPages, setTotalPages] = useState(0);
-
   const [isLoading, setIsLoading] = useState(false);
   
   // 12 Cards per page
   const CARD_LIMIT: number = 12;
   const SEARCH_DELAY: number = 300;
 
-  useEffect(() => {
+  // State variables to see if search value changes (useful when user changes a filter past page 1)
+  const [prevSearch, setPrevSearch] = useState<string | undefined>(undefined);
```

Comment:
I think you can remove the `undefined` from these states and just have: `const [prevSearch, setPrevSearch] = useState<string>(''); `. 
This should initialize the string as just an empty string, instead of undefined, at the start. But this also means that  the `search`, `sortBy`, and `order` state variables must always be strings. So, you should also change their variable types in the props interface above.

Reply:
Ok sounds good I will make this change.